# planner setup
# maximum number of agents
NumberofAgents: 2